corehq/ex-submodules/pillowtop/tests/test_bulk.py

```diff
@@ -52,7 +52,7 @@ def test_get_errors_with_ids(self):
 
 
 @sharded
-@es_test(index=TEST_INDEX_INFO)
+@es_test
```

Comment:
What does this decorator do in these tests?

Reply:
It adds the `es_test` nose attribute, so you can run all decorated tests like this:

```shell
./manage.py test --attr=es_test
```

Reply:
Ohh! Got it. Thanks 👍